pkg/hintrunner/hint_test.go

```diff
@@ -268,6 +268,110 @@ func TestWideMul128(t *testing.T) {
 	)
 }
 
+func TestUint256DivMod(t *testing.T) {
+	vm := defaultVirtualMachine()
+	vm.Context.Ap = 0
+	vm.Context.Fp = 0
+
+	var quotient0 ApCellRef = 1
+	var quotient1 ApCellRef = 2
+	var remainder0 ApCellRef = 3
+	var remainder1 ApCellRef = 4
+
+	dividend0Felt := f.NewElement(89)
+	dividend1Felt := f.NewElement(72)
+
+	divisor0Felt := f.NewElement(3)
+	divisor1Felt := f.NewElement(7)
```

Comment:
It would also be nice to test an edge case as well, for example the division between 256 bit numbers, I mean, so they cannot fit in a felt once you put the two parts together